lib/actions/reject.js

```diff
@@ -9,6 +9,7 @@ var __awaiter = (this && this.__awaiter) || function (thisArg, _arguments, P, ge
     });
 };
 Object.defineProperty(exports, "__esModule", { value: true });
+exports.reject = void 0;

```

Comment:
Make sure you update the `typescript` version in the package.json file to avoid this later